docs/dictionary/command/libURLSetStatusCallback.lcdoc

```diff
@@ -33,7 +36,10 @@ The name of a message to be sent whenever the URLStatus function is
 updated. 
 
 objectLongID:
-The long ID of the object to send the message to.
+The long ID of the object to send the message to. If <messageName> is 
+not empty but <objectLongID> is empty then the message will be sent to
+the revLibURL library stack. The message can be handled in a frontscript,
+backscript, or library stack.
```

Comment:
@trevordevore I'm really sorry, but I missed this: I think this needs to be changed from:

> If <messageName> is not empty but <objectLongIDE> is empty

to:

> If <messageName is not empty and <objectLongID> is not specified

in order to match my suggested changes to how `libURLSetStatusCallback` handles the arguments.

I'm sorry for not spotting this earlier! 😞